lib/driverProviders/direct.js

```diff
@@ -51,18 +51,16 @@ DirectDriverProvider.prototype.getNewDriver = function() {
   var driver;
   switch (this.config_.capabilities.browserName) {
     case 'chrome':
-      var chromeDriverFile = this.config_.chromeDriver ||
-          path.resolve(__dirname, '../../selenium/chromedriver');
+      var defaultChromeDriverPath = path.resolve(__dirname, '../../selenium/chromedriver');
 
-      // Check if file exists, if not try .exe or fail accordingly
-      if (!fs.existsSync(chromeDriverFile)) {
-        chromeDriverFile += '.exe';
+      if (process.platform.indexOf('win') === 0)
```

Comment:
Style nitpick, we use brackets in if statements in the rest of the project (it looks like we don't have that in the jshintrc, i'll add a PR to fix that)